### ISSUE TITLE
Beautify removed last "end"

### ISSUE THREAD

**Issue by gneiss01** [erniebrodeur/ruby-beautify#23]
Hi,

After trying to beautify my codes, it introduced lot of errors and one of these is:

Original:
module MyModule::Exceptions
    class NotFoundError < StandardError
    end
end

After executing "ruby-beautify -c 2 -s --overwrite notfounderror.rb" it became
module MyModule::Exceptions
  class NotFoundError < StandardError
  end

Am I missing something in the command line options?
Thanks,

**Comment by erniebrodeur** [comment]
No, it has a few bugs, I'll try to figure out the cause of this one and get it fixed.

One of them appears to be stripping trailing end.

**Comment by erniebrodeur** [comment]
IT only strips trailing ends under certain cases, i don't entirely understand.  Need to check my crawler to see whats going on in more detail.

**Comment by opsb** [comment]
It looks like it might be if there's a \n missing at the end of the file.

**Comment by erniebrodeur** [comment]
I've seen it too, I think that's correct, I'll make a scenario for my tests soon and get this patched.

**Comment by hzulla** [comment]
fixed in #31
